src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -108,13 +165,13 @@ protected override void BeginProcessing()
                 }
             }
             catch (Exception e) when (
+
                 // Handle exceptions related to file access to provide more specific error message
                 // https://docs.microsoft.com/en-us/dotnet/standard/io/handling-io-errors
                 e is IOException ||
                 e is UnauthorizedAccessException ||
                 e is NotSupportedException ||
-                e is SecurityException
-            )
+                e is SecurityException)
```

Comment:
Please revert this style change.

Reply:
This is due to CodeFactor I believe.

Reply:
I see. But please remove the blank line above the comment. It's OK that CodeFactor will complain about it, it doesn't block the CI.

Reply:
@daxian-dbw Thanks for clarification. I have removed this.